client/src/components/History/Export/RDMCredentialsInfo.vue

```diff
@@ -0,0 +1,23 @@
+<script setup lang="ts">
+import { BAlert } from "bootstrap-vue";
+import { RouterLink } from "vue-router";
+
+interface Props {
+    selectedRepository?: string;
+    what?: string;
+}
+
+withDefaults(defineProps<Props>(), {
+    selectedRepository: "the selected repository",
+    what: "file",
+});
+</script>
+
+<template>
+    <BAlert show variant="info">
+        You may need to setup your credentials for {{ selectedRepository }} in your
+        <RouterLink to="/user/information" target="_blank">preferences page</RouterLink> to be able to export. You can
+        also define some default options for the export in those settings, like the public name you want to associate
+        with your records or whether you want to publish them immediately or keep them as drafts after export.
+    </BAlert>
```

Comment:
This is more accurate now. The publishing can only be handled from the Zenodo website for extra security.

```suggestion
        with your records.
```